readme.md

```diff
@@ -240,6 +240,7 @@ exports.profanitySureness = Math.floor(Math.random() * 3)
 ```
 
 The `allow` field should be an array of rules (the default is `[]`).
+This array should contain rule identifiers, not individual words. Rule identifierss can be found in the [retext.js](https://github.com/retextjs) rules files, [https://github.com/retextjs/retext-equality/blob/master/rules.md](retextjs/retext-equality) and [https://github.com/retextjs/retext-profanities/blob/master/rules.md](retxtjs/retext-profanities).
```

Comment:
```suggestion
Note that words don’t work; use rule identifiers instead (those shown next to
warnings).
```

Thanks for the clarification Dustin!
I find your notes a bit verbose: there are already several places that mention rules and link to the retext projects, so I’m proposing the above.

Btw, two of your links aren’t links, see https://commonmark.org/help/

And finally, we check prose and markdown in this project, which is why Travis fails. See [Contributing](https://github.com/get-alex/alex#contribute) for help.